pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -354,7 +355,7 @@ func (c *Controller) syncService(key string) error {
 
 	esLabelSelector := labels.Set(map[string]string{
 		discovery.LabelServiceName: service.Name,
-		discovery.LabelManagedBy:   controllerName,
+		discovery.LabelManagedBy:   c.reconciler.GetControllerName(),
```

Comment:
Can use `controllerName` directly here. Would eliminate the need for the `GetControllerName` function.

Reply:
I want to expose controllerName.